src/common/syncjournaldb.cpp

```diff
@@ -17,21 +17,21 @@
  */
 
 #include <QCryptographicHash>
+#include <QDir>
+#include <QElapsedTimer>
 #include <QFile>
 #include <QLoggingCategory>
 #include <QStringList>
-#include <QElapsedTimer>
 #include <QUrl>
-#include <QDir>
-#include <sqlite3.h>
 #include <cstring>
+#include <sqlite3.h>
```

Comment:
@TheOneRing re https://github.com/owncloud/client/pull/9044#issuecomment-921629336 this is:
https://github.com/jsoref/owncloud-client/commit/51b6bb7eda59ad346336f5010ff867f2fe1171ad

I'd encourage you to merge that commit outside of this PR, since it really doesn't belong, but the linter failed and it was way too messy for me to handle it any other way here. -- And the only way to show it mattered was to compose this PR.